lib/generateNormals.js

```diff
@@ -127,8 +128,23 @@ function generateFaceNormals(gltf, primitive, positionSemantic) {
                     Cartesian3.subtract(positionTwo, positionOne, scratchNormalOne);
                     Cartesian3.subtract(positionThree, positionOne, scratchNormalTwo);
                     var normal = new Cartesian3();
-                    Cartesian3.cross(scratchNormalOne, scratchNormalTwo, normal);
-                    Cartesian3.normalize(normal, normal);
+
+                    // Check for degenerate triangles
```

Comment:
When this is merged, please update https://github.com/AnalyticalGraphicsInc/gltf-pipeline/issues/242 to note that this check is here.